code_conventions.md

```diff
@@ -107,6 +107,14 @@ public:
 };
 ```
 
+### Solution for hotend versus nozzle and {}_index versus {}_nr or {}_number name usage:
```

Comment:
In Cura we use 'extruder', when this was brought up during the Arcitecture Guild meeting we where told that this was ok as well for us (as in, this was for Embedded)?

Reply:
Generally, extruder is the considered the whole thing, feeder+bowdentube+hotend. In embedded we see the hotend and feeder as different things, and thus using the name extruder we avoid, as this can cause confusion.

Reply:
Ok, makes sense :-)